hermetic/src/cli/tester.rs

```diff
@@ -57,10 +57,11 @@ pub async fn run(opts: TestOpts) -> Result<()> {
     // Parse network file
     let mut network: Network = serde_yaml::from_str(&fs::read_to_string(&opts.network).await?)?;
     debug!("input network {:#?}", network);
-    network.spec.ceramic.push(CeramicSpec {
-        ipfs: Some(IpfsSpec::Rust(RustIpfsSpec::default())),
-        ..Default::default()
-    });
+
+    // If the test driver fails for any reason the network will not be cleaned up. We set an expiration for the network
```

Comment:
This comment is now misleading in that the driver never cleans up networks. Instead it always relies on the ttl